js/templates/modals/baseModal.html

```diff
@@ -1,4 +1,22 @@
+<div class="popInMessageHolder js-popInMessages"></div>
 <div class="<%= ob.modelContentClass %>">
-  <i class="jsModalClose <%= ob.closeButtonClass %> <% !ob.showCloseButton && print('hide') %>"></i>
+  <div class="topControls pureFlex spread js-closeClickTarget js-btcTicker">
```

Comment:
I'm not sure how I feel about putting this section in the baseModal. As we add new modals, this could become very long. What's the reason to put all of the view-specific markup in the base template?